test/browser/test.js

```diff
@@ -24,10 +24,8 @@
       });
```

Comment:
I wish I had a better understanding of how GitHub decided certain files are autogenerated and certain files aren't :p